backend/src/models/jobs.go

```diff
@@ -74,7 +74,7 @@ const (
 	SyncVideoMetadataJob   JobType   = "sync_video_metadata"
 	PutVideoMetadataJob    JobType   = "put_video_metadata"
 	AddVideosJob           JobType   = "add_videos"
-	EveryThreeHours        string    = "0 */3 * * *"
+	EveryThreeHours        string    = "0 6-20 * * *"
```

Comment:
Since the cron changed to every hour vs every 3 hours, should the variable name be changed to reflect this?